### PR TITLE
[Refactor] Use promises in 'SourceCodeModule'

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

**Comment by satya164** [comment]
cc @dmmiller

File: ReactAndroid/src/main/java/com/facebook/react/modules/debug/SourceCodeModule.java [hunk 1]
```diff
@@ -35,14 +35,14 @@ public SourceCodeModule(String sourceUrl, String sourceMapUrl) {
 
   @Override
   public String getName() {
-    return "RKSourceCode";
+    return "RCTSourceCode";
```
Comment (satya164) [inline ReactAndroid/src/main/java/com/facebook/react/modules/debug/SourceCodeModule.java]: I'm not sure about this. I searched the codebase for `RKSourceCode` and it's not used anywhere. `RCTSourceCode` is used in JavaScript and Obj-C, but it's not there in any Java code.

**Comment by mkonicek** [comment]
Nice! I'll let @dmmiller review.

File: ReactAndroid/src/main/java/com/facebook/react/modules/debug/SourceCodeModule.java (hunk 1 above)
Comment (dmmiller) [inline ReactAndroid/src/main/java/com/facebook/react/modules/debug/SourceCodeModule.java]: There is code somewhere that does translation.  This is probably fine.

**Comment by dmmiller** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**